Polyfills/Canvas/Source/Canvas.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <Babylon/Polyfills/Canvas.h>
+#include <GraphicsImpl.h>
+
+namespace Babylon::Polyfills::Internal
+{
+    class NativeCanvas final : public Napi::ObjectWrap<NativeCanvas>
+    {
+    public:
+        using ParentT = Napi::ObjectWrap<NativeCanvas>;
```

Comment:
Why is this public?

Reply:
Moved.

Reply:
why do we define this ParentT type?

Reply:
it feels like we don't do this elsewhere/having a custom type for a commonly used type may generate more confusion than worth